llvm/lib/Bitcode/Writer/BitcodeWriter.cpp

```diff
@@ -4723,6 +4725,8 @@ static const char *getSectionNameForCommandline(const Triple &T) {
     return ".llvmcmd";
   case Triple::XCOFF:
     llvm_unreachable("XCOFF is not yet implemented");
+  case Triple::Repo:
```

Comment:
I think this needs a break before it.

Reply:
Since the  `llvm_unreachable("XCOFF is not yet implemented");` is ended with `abort()`, it will abort the current process. Therefore, this doesn't need a break before it.